examples/registration_form/lib/root/root_component.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:angular/angular.dart';
+import 'package:angular_components/angular_components.dart';
+import 'package:angular_forms/angular_forms.dart';
+import 'package:examples.registration_form/address/address_component.dart';
+
+@Component(
+    selector: 'hello-world',
+    templateUrl: 'root_component.html',
+    directives: [
+      formDirectives,
+      materialInputDirectives,
+      MaterialYesNoButtonsComponent,
+      MaterialSaveCancelButtonsDirective,
+      NgIf,
+      AddressComponent
+    ],
+    providers: [
+      popupBindings,
+    ])
+class RootComponent {
+  @ViewChild('myForm')
+  ControlContainer myForm;
+
+  String firstName;
+  String lastName;
```

Comment:
I'd drop these they aren't needed. Maybe keep one if you want to show ngModel usage, but since you aren't using the values they just add a bit of noise.

Reply:
Done